### ISSUE TITLE
We still need to fix tab bar buttons themselves

### ISSUE THREAD

**Issue by 117649** [onemen/TabMixPlus#33]
2 out of 3 not working right.

**Comment by onemen** [comment]
I will do it after #21

**Comment by onemen** [comment]
Fixed by https://github.com/onemen/TabMixPlus/commit/592f36e8e55000d5099056eb781ad4f88373ef8b and https://github.com/onemen/TabMixPlus/commit/5a852c1979a1d025d6efef4b9f25cf9b384b0904

**Comment by 117649** [comment]
![image](https://user-images.githubusercontent.com/8281345/108586275-48428e80-7388-11eb-9649-cd8b968060f2.png)
In overflow menu TMP's menu-button are treated as button. I think there may be more to discuss.  
Newer type of items that replaced menu-button are behave as a submenu in that menu.(That 'Profiler' there is a good example)

**Comment by onemen** [comment]
test it with the latest fix https://github.com/onemen/TabMixPlus/commit/1cbc558a329d8109dcdd70c2b3fc25df749705d2

**Comment by 117649** [comment]
> test it with the latest fix [1cbc558](https://github.com/onemen/TabMixPlus/commit/1cbc558a329d8109dcdd70c2b3fc25df749705d2)

It is working. 

By the way, separator between page list and option button display a bright white in overflow menu while using dark theme. but on tab bar it using a darkish gray.
![image](https://user-images.githubusercontent.com/8281345/108616745-4d1c4680-744b-11eb-9943-719081cee3a6.png)
![image](https://user-images.githubusercontent.com/8281345/108616795-d03d9c80-744b-11eb-8453-2af2e18cc208.png)
This is just a minor thing, just for the record.

**Comment by onemen** [comment]
I will edit general.css

```diff
  .tabmix-subview menuseparator.subviewbutton {
+   border-top: 1px solid var(--panel-separator-color);
    min-height: 0;
    padding: 0;
  }
```
